DateTimePicker/DateTimePicker.js

```diff
@@ -64,15 +81,15 @@ const DateTimePickerBase = kind({
 		className: 'dateTimePicker'
 	},
 
-	render ({disabled, onChange, value, ...rest}) {
+	render ({disabled, onChange, spotlightDisabled, value, ...rest}) {
 
 		return (
 			<Row {...rest} className={css.dateTimePicker} align="center center">
 				<Row align="center center">
-					<TimePicker css={css} disabled={disabled} onChange={onChange} value={value} />
+					<TimePicker css={css} disabled={disabled} onChange={onChange} spotlightDisabled={spotlightDisabled} value={value} />
```

Comment:
Shouldn't `onSpotlightDisappear` be here and there, also?

Reply:
upps. fixed